lib/views_d/mein_profil.dart

```diff
@@ -174,18 +176,63 @@ class _MeinProfilDState extends State<MeinProfilD> {
                                 ),
                               )),
                         ),
-                      ]))
-            ]),
-      bottomNavigationBar: BottomMenu(index: 0),
-    );
+                        Padding(
+                          padding: const EdgeInsets.all(15.0),
+                          child: SizedBox(
+                            width: MediaQuery.of(context).size.width * 0.9,
+                            child: Card(
+                                clipBehavior: Clip.antiAlias,
+                                shape: RoundedRectangleBorder(
+                                  borderRadius: BorderRadius.circular(20),
```

Comment:
Ich würde vorschlagen, dass die Netiquette und die Trophäen den gleichen BorderRaduis haben sollten und der Text bei der Netiquette auch mittig ist. Ist aber auch nur ein Vorschlag, denke aber, dass es einheitlicher sein könnte. Vll könnte man den Text "Netiquette" auch einfach in einem Grauton machen, damit klar erkennbar ist, dass es eher zweitrangig ist und nur eine Info auf der Seite ist.

Reply:
Border Radius wird problematisch wenn man es ausklappt. würde aber mal mit aufnehmen